motionplan/plannerOptions.go

```diff
@@ -194,26 +194,45 @@ type PlannerOptions struct {
 	// See metrics.go for options
 	ConfigurationDistanceMetric ik.SegmentFSMetricType `json:"configuration_distance_metric"`
 
+	// A profile indicating which of the tolerance parameters listed below should be considered
+	// for further constraining the motion.
 	MotionProfile MotionProfile `json:"motion_profile"`
 
+	// Linear tolerance for translational deviation for a path. Only used when the
+	// `MotionProfile` is `LinearMotionProfile`.
 	LineTolerance float64 `json:"line_tolerance"`
 
+	// Orientation tolerance for angular deviation for a path. Used for either the `LinearMotionProfile`
+	// or the `OrientationMotionProfile`.
 	OrientationTolerance float64 `json:"orient_tolerance"`
 
+	// A factor by which the entire pose is allowed to deviate for a path. Used only for a PseudolinearMotionProfile.
 	ToleranceFactor float64 `json:"tolerance"`
 
+	// CollisionBufferMM dictates how far the robot should be from any obstacle during its motion.
```

Comment:
I'd prefer a more specific "No two geometries that did not start the motion in collision may come within this distance of one another at any time during a motion"